src/components/ProductTableCard/index.tsx

```diff
@@ -0,0 +1,62 @@
+import { useTranslation } from 'react-i18next';
+
+import { Box, Typography } from '@mui/material';
+
+import { IProduct } from 'src/redux/product/types';
+import theme from 'src/theme';
+
+import { Image, ImageWrapper } from './styles';
+
+type PartialProduct = Partial<Pick<IProduct, 'name' | 'size' | 'images'>>;
+
+function ProductTableCard({ images, size, name }: PartialProduct) {
+  const { t } = useTranslation();
+
+  return (
+    <Box display="flex" gap="16px" alignItems="center">
+      <ImageWrapper>
+        {images && images.length > 0 ? (
+          <Image
+            width="52px"
+            height="67px"
+            src={images[0]}
+            alt={name || 'Product image'}
+          />
+        ) : (
+          <Image
+            width="52px"
+            height="67px"
+            src="src/assets/photos/no-image.jpg"
+            alt="Default image"
+          />
+        )}
```

Comment:
the difference in 2 images only on url, or condition to url not to full component

Reply:
thank you, fixed it